.coveragerc

```diff
@@ -0,0 +1,18 @@
+# this file is based on the examples provided on scikit-learn's .coveragerc
```

Comment:
are you sure that we need to have this file at all? Do other biocore repos have this file?